calcom/cal.com
# perf: Remove duplicate call for user event types

## What does this PR do?

Removes a duplicate call to get all event types linked directly to a user.

## Type of change




## How should this be tested?

- Ensure that all event types created by the user load
- Ensure managed event types load for users

## Mandatory Tasks
